pkg/controller/v1alpha1/inferencegraph/knative_reconciler.go

```diff
@@ -170,6 +171,18 @@ func createKnativeService(componentMeta metav1.ObjectMeta, graph *v1alpha1api.In
 			},
 		},
 	}
+
+	// Only adding this env variable "PROPAGATE_HEADERS" if router's headers config has the key "propagate"
+	value, exists := config.Headers["propagate"]
+	if exists {
+		service.Spec.ConfigurationSpec.Template.Spec.PodSpec.Containers[0].Env = []v1.EnvVar{
+			{
+				Name:  "PROPAGATE_HEADERS",
```

Comment:
Can we define this as a constant as it is used in multiple places

Reply:
Addressed